scripts.js

```diff
@@ -0,0 +1,66 @@
+let myLibrary = [];
+
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
+}
+
+function checkBooks(library) {
+  return library.length === 0 ? 'There are no books' : '';
+}
+
+function render() {
+  const bookList = document.getElementById('book-list');
+  bookList.innerHTML = checkBooks(myLibrary);
+
+  myLibrary.forEach((element, index) => {
+    const row = document.createElement('tr');
+    row.id = index;
+    const checked = element.read ? 'checked' : '';
+
+    row.innerHTML = `<td>${element.title}</td> <td>${element.author}</td>\n <td>${element.pages}</td> <td><input onclick="updateBook(${index})" type="checkbox" name="read" id="read" ${checked}></td>\n`;
+    row.innerHTML += `<td><button onclick="removeBook(${index})"=>Delete Book</button></td>`;
+    bookList.appendChild(row);
+  });
+}
+
+function addBookToLibrary() { // eslint-disable-line no-unused-vars
+  const title = document.getElementById('title').value;
```

Comment:
- [ ] [OPTIONAL] It could be a good idea to first select the elements by id in the DOM and then in another line grab the values after checking if the element was indeed found. 
That would avoid any error in case some of the DOm is not rendered in case of a bug or similar.